dubbo-common/pom.xml

```diff
@@ -77,5 +77,9 @@
             <groupId>de.ruedigermoeller</groupId>
             <artifactId>fst</artifactId>
         </dependency>
+        <dependency>
```

Comment:
dubbo-common is the most basic module that all the other modules might depend on, I think it should be depend on nacos-client here. It looks weird.